lib/openstax/rescue_from/engine.rb

```diff
@@ -0,0 +1,21 @@
+require 'active_support/all'
+require 'exception_notification'
+
+require "openstax/rescue_from/configuration"
+require "openstax/rescue_from/version"
+
+module OpenStax
+  module RescueFrom
+    class Engine < ::Rails::Engine
+      initializer 'openstax.rescue_from.inflection' do
+        ActiveSupport::Inflector.inflections do |inflect|
+          inflect.acronym 'OpenStax'
```

Comment:
I could be wrong but I don't think you need this unless using Rails to autoload stuff in the gem. And `openstax_utilities` already does this for you.